tensorboard/BUILD

```diff
@@ -1,12 +1,12 @@
 # Description:
 # TensorBoard, a dashboard for investigating TensorFlow
 
-load("//tensorboard/defs:py_repl.bzl", "py_repl")
-load("//tensorboard/defs:web.bzl", "tf_web_library")
-load("//tensorboard/defs:zipper.bzl", "tensorboard_zip_file")
 load("@rules_python//python:py_binary.bzl", "py_binary")
 load("@rules_python//python:py_library.bzl", "py_library")
 load("@rules_python//python:py_test.bzl", "py_test")
+load("//tensorboard/defs:py_repl.bzl", "py_repl")
```

Comment:
This (and other files) were not in the original CL. Is this from your buildifier run? If so, which version of buildifier are you using? CI uses 3.0.0. Looks like this is 3.0.0 compatible, though (but so was the old order).

Reply:
Ah, I just ran the command, hoping that my machine would find it, but it looks like it used an internal, more recent version.

I used the `download_buildifier.sh` script from our repo, just like the CI workflow does, to get that same version, reverted the previous commit, and ran that command from the CI workflow, without the `--mode=check` flag.

For future reference, is there an easier/better way to accomplish this?

Reply:
I don't know if there is an easier/better way to accomplish. I basically did/do the same as you describe.